.travis.yml

```diff
@@ -15,7 +15,6 @@ install:
 script:
   - set -e
   - make
-  - make test-e2e TEST_IMAGE_REPO="$(./hack/install-registry.sh show):5000/shipwright-io/build-e2e"
```

Comment:
@SaschaSchwarze0 I assume we'll remove Travis in a follow-up PR?

Reply:
I was thinking the same. There is only the simple `make` remaining. I am fine to remove the entire `.travis.yml` if you agree. Can do this in this PR.

Reply:
Removed it now, there is really no value to keep it.